server/modules/module_def.js

```diff
@@ -112,13 +113,14 @@ export class ModuleDef extends EventEmitter {
     };
   }
   // Returns a new module def that extends this def with new configuration.
-  extend(name, config, credit) {
+  extend(name, config, credit, exclude) {
     return new ModuleDef(
       name,
       this.root,
       {base: this},
       config,
       credit,
+      exclude,
```

Comment:
Just throwing it out there: "testonly" ?

Reply:
I'm going to do this for the lols.